File: coinkiri-core/src/main/kotlin/coinkiri/core/domain/coin/repository/CoinRepository.java
package coinkiri.core.domain.coin.repository;

import coinkiri.core.domain.coin.Coin;
import org.springframework.data.jpa.repository.JpaRepository;

/**
 * @ClassName    : CoinRepository
 */
public interface CoinRepository extends JpaRepository<Coin, Long>, CoinRepositoryQDsl {


}

File: coinkiri-core/src/main/kotlin/coinkiri/core/domain/coin/repository/CoinRepository.kt
package coinkiri.core.domain.coin.repository

import coinkiri.core.domain.coin.Coin
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

/**
 * @ClassName    : CoinRepository
 */
interface CoinRepository : JpaRepository<Coin, Long> {

}
